content/aips/C21F601D0D3F-AIP-STKAAVE_RETRIEVAL.md

```diff
@@ -0,0 +1,75 @@
+---
+
```

Comment:
Rename the file from `C21F601D0D3F-AIP-STKAAVE_RETRIEVAL.md` to `C21F601D0D3F-AIP-STKAAVE-RETRIEVAL.md`